smart_card_connector_app/src/usb-devices-logging.js

```diff
@@ -0,0 +1,104 @@
+/** @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview This script does some minor device logging on the USB level
+ * (therefore it sees some stuff that can miss the higher PC/SC level).
+ */
+
+goog.provide('GoogleSmartCard.ConnectorApp.setupUsbDevicesLogging');
+
+goog.require('GoogleSmartCard.DebugDump');
+goog.require('GoogleSmartCard.Logging');
+goog.require('goog.array');
+goog.require('goog.log.Logger');
+
+goog.scope(function() {
+
+/** @const */
+var USB_DEVICE_FILTERS = [{'interfaceClass': 0x0B}];
```

Comment:
This duplicates the constant in window-devices-displaying.js. In order to rid of copy-pasting, the constant may be removed from this file - it's not crucially important for the logging stuff.